micro-rdk/src/common/app_client.rs

```diff
@@ -218,6 +227,27 @@ impl AppClient {
         Ok(CertificateResponse::decode(r)?)
     }
 
+    pub(crate) fn initiate_signaling(
```

Comment:
This no longer awaits completion of a call because of the long polling nature of `/Answer` as discussed in https://viam.atlassian.net/issues/RSDK-7575